src/main/resources/db/migration/dev/V1_7__create_organization_table.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE public.organizations (
+    id UUID PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    address VARCHAR(255),
+    note VARCHAR(255)
+);
+
+CREATE TABLE public.user_in_organization (
+    id UUID PRIMARY KEY,
+    user_id UUID,
+    organization_id UUID,
+    organization_permission VARCHAR(255) ARRAY,
+    CONSTRAINT fk_user_user_in_org FOREIGN KEY (user_id) REFERENCES users(id),
+    CONSTRAINT fk_org_user_in_org FOREIGN KEY (organization_id) REFERENCES organizations(id)
+);
+
+CREATE TABLE resource_in_organization (
```

Comment:
what is the difference between public.<tableName> and just <tableName>? Does it make any difference?